### PR TITLE
Add post-human cyborg profession to mod.

### PR DESCRIPTION
**Pull request by Noctifer-de-Mortem**
Implementation of #18658 to the More Professions and Scenarios mod rather than to core.

Same items and bionics but raised cost to 8 as suggested by Coolthulhu.

### CODE REVIEW

**Comment by DangerNoodle** [comment]
Interesting. Are all of these passive bionics? Because no power or power generation is included.

File: data/mods/more_classes_scenarios/cs_classes.json [hunk 1]
```diff
@@ -155,5 +155,46 @@
             ]
         },
         "flags" : ["SCEN_ONLY"]
+    },
+    {
+        "type" : "profession",
+        "ident": "trans_human",
+        "name": "Post-Human Cyborg",
+        "description": "As a welthy trans-humanist you decided to put yourself on the frontline of augemetive technology to bring forth the future. You are now a walking example of what humanity could of had become.",
```
Comment (shedya) [inline data/mods/more_classes_scenarios/cs_classes.json]: wealthy, augmentative, could had (without of)

Comment (DangerNoodle) [inline data/mods/more_classes_scenarios/cs_classes.json]: Or could have, or could've, etc.

File: data/mods/more_classes_scenarios/cs_classes.json [hunk 2]
```diff
@@ -155,5 +155,46 @@
             ]
         },
         "flags" : ["SCEN_ONLY"]
+    },
+    {
+        "type" : "profession",
+        "ident": "trans_human",
+        "name": "Post-Human Cyborg",
+        "description": "As a wealthy trans-humanist you decided to put yourself on the frontline of augmentative technology to bring forth the future. You are now a walking example of what humanity could had become.",
```
Comment (illi-kun) [inline data/mods/more_classes_scenarios/cs_classes.json]: It's sad I need to repeat it once more: use double space after period.

**Comment by mugling** [comment]
> It's sad I need to repeat it once more: use double space after period.

Ok lets add this to linting and discussed by @codemime. Possibly +/- spell checking for textual fields. How to avoid false positives?

**Comment by codemime** [comment]
> Ok lets add this to linting and discussed by @codemime. Possibly +/- spell checking for textual fields. How to avoid false positives?

That discussion is in #18665.

**Comment by illi-kun** [comment]
I saw the discussion. I have no strong opinion about auto-check of double space but it should be nearly 100% effective to be merged. I disagree with spellcheck because I tested some spellcheckers previously on our texts and the have false positive cases. I can provide expanded feedback on this later, when I will be in front of my laptop.

**Comment by mugling** [comment]
> I have no strong opinion about auto-check of double space but it should be nearly 100% effective to be merged

That's the difficulty - false positives are especially nasty in automated tools

File: data/mods/more_classes_scenarios/cs_classes.json [hunk 3]
```diff
@@ -155,5 +155,46 @@
             ]
         },
         "flags" : ["SCEN_ONLY"]
+    },
+    {
+        "type" : "profession",
```
Comment (mugling) [inline data/mods/more_classes_scenarios/cs_classes.json]: Not essential here but try for consistent JSON formatting

**Pull request merged**